src/db.js

```diff
@@ -1,18 +1,31 @@
 const flatfile = require('flat-file-db')
 const promise = require('promise')
 
-const db = flatfile.sync(process.env.DB_NAME || 'views.db')
+let adapter
 
-const promisifiedDb = {
-  // Promisify async operations
-  put: promise.denodeify(db.put.bind(db)),
-  del: promise.denodeify(db.del.bind(db)),
-  clear: promise.denodeify(db.clear.bind(db)),
+const repeatCharacter = (char, n) => `${Array(n + 1).join(char)}`
 
-  get: db.get.bind(db),
-  has: db.has.bind(db),
-  keys: db.keys.bind(db),
-  close: db.close.bind(db),
+if (process.env.DB_ADAPTER) {
+  const adapterName = `micro-analytics-adapter-${process.env.DB_ADAPTER}`
+  try {
+    adapter = require(adapterName)
+    console.log(`Using ${adapterName}`)
+  } catch (err) {
+    if (err.code === 'MODULE_NOT_FOUND') {
+      // Console.error a warning message, but normally exit the process to avoid printing ugly npm ERR lines and stack trace.
+      console.error(`\n${repeatCharacter(' ', 22)}⚠️ ERROR ⚠️\n${repeatCharacter('-', 55)}\nYou specified "${process.env.DB_ADAPTER}" as the DB_ADAPTER, but no package\ncalled "${adapterName}" was found.\n\nPlease make sure you spelled the name correctly and\nhave "npm install"ed the necessary adapter package!\n${repeatCharacter('-', 55)}\n`)
+      process.exit(0)
+    }
+  }
+} else {
+  console.log('Using standard flat-file-adapter')
```

Comment:
I think the idea was to match the fact that it [logs out](https://github.com/mxstbr/micro-analytics/pull/14/files#diff-f786454c0651c074b51fa96e1b531414R12) the overridden adapter.